master/internal/trial.go

```diff
@@ -401,6 +407,22 @@ func (t *trial) runningReceive(ctx *actor.Context) error {
 	case scheduler.ContainerStarted:
 		t.containers[msg.Container.ID()] = msg.Container
 
+		names := make([]string, 0, len(msg.Container.Addresses()))
```

Comment:
question: do we ever use proxying for anything in trial?

Reply:
removed that. It seems fine :)